thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-comparison-range/component.js

```diff
@@ -9,12 +10,29 @@ import moment from 'moment';
  */
 const serverDateFormat = 'YYYY-MM-DD HH:mm';
 
+/**
+ * @summary Mapping between values that are named on the backend to epoc time value
+ * @type {Object}
+ * @example  `1_HOURS=3600=1hr , 1800=30mins` etc since we using epoc time values
+ */
+const namedToEpocMapping = {
```

Comment:
moment js should be able to handle this generically.

Reply:
True. I might have to refractor this when i do the componentize/service in the next PR. For now i want to get it out to be used and tested.